nemo_skills/dataset/numb3rs/prepare.py

```
@@ -0,0 +1,310 @@
+def build_messages_with_prompt(audio_metadata, prompt_text):
+    """Build OpenAI format messages with a specific prompt and audio."""
+    system_message = {"role": "system", "content": SYSTEM_MESSAGE}
+    user_message = {
+        "role": "user",
+        "content": prompt_text,
+        "audio": audio_metadata,
+    }
+    return [system_message, user_message]
+
+
+def save_audio_and_format_entry(entry, category, audio_dir, sample_idx, with_audio=True, audio_prefix="/data/numb3rs"):
```

Ruff (0.14.14)

[warning] Unused function argument: `sample_idx`

(ARG001)